tutoraspects/templates/aspects/apps/aspects/migrations/alembic/versions/0036_course_enrollments.py

```diff
@@ -0,0 +1,39 @@
+from alembic import op
+import sqlalchemy as sa
+
+revision = "0036"
+down_revision = "0035"
+branch_labels = None
+depends_on = None
+on_cluster = " ON CLUSTER '{{CLICKHOUSE_CLUSTER_NAME}}' " if "{{CLICKHOUSE_CLUSTER_NAME}}" else ""
+engine = "ReplicatedMergeTree" if "{{CLICKHOUSE_CLUSTER_NAME}}" else "MergeTree"
+
+
+def upgrade():
+    op.execute(
+        f"""
+        -- Raw table that Ralph writes to
+        CREATE TABLE IF NOT EXISTS {{ ASPECTS_EVENT_SINK_DATABASE }}.course_enrollment
+        {on_cluster}
+        (
+            id Int32 NOT NULL,
+            course_key String NOT NULL,
+            created String NOT NULL,
+            is_active Bool NOT NULL,
+            mode LowCardinality(String) NOT NULL,
+            username String NOT NULL,
+            user_id Int32 NOT NULL,
+            dump_id UUID NOT NULL,
+            time_last_dumped String NOT NULL
+        ) ENGINE {engine}
+        ORDER BY (course_key, id, time_last_dumped)
```

Comment:
Is there a reason to use `id` here instead of `user_id`? I would expect that in most cases we'd be querying this by course & user, but I don't think we'd ever care about the Django PK.

Reply:
Correct, that order key is wrong